### PR TITLE
Add jit-saving to the crop classifier

### LINKED ISSUE #2: `torch.jit` for the crop classifier
To better integrate this into our inference pipeline, save the model as a `torch.jit` model.

- [ ] Function to save the model after finetuning
- [ ] Test to ensure the `jit` model has the same outputs as the un`jit`ted model

cc @ivanzvonkov

### PR DESCRIPTION
**Pull request by gabrieltseng**
- [x] Save the classifier as a `jit` model
- [x] Tests to make sure it works

Closes #2

### CODE REVIEW

File: crop_classification/tests/test_lstm.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    model.save("timl", tmp_path)
+
+    loaded_model = torch.jit.load(tmp_path / f"timl.pt")
+    loaded_model.eval()
+
+    x = torch.rand(5, num_timesteps, input_size)
+
+    with torch.no_grad():
+        loaded_y = loaded_model(x).numpy()
+        org_y = model(x).numpy()
```
Comment (ivanzvonkov) [inline crop_classification/tests/test_lstm.py]: `y_from_jit_model` and `y_from_state_dict_model` might be more clear

**Comment by ivanzvonkov** [comment]
I think it would also be beneficial to have a test to verify that new model results are within the scope of reality. In crop-mask I run each model on its own validation set as part of an integration set. This is probably best left for a separate PR but just thought I'd mention it now

**Review by ivanzvonkov — APPROVED** [review]
Looks good!
